src/go/k8s/tests/e2e-v2/node-deleted/03-assert.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+- command: ../../../hack/get-redpanda-info.sh redpanda ../../_e2e_artifacts_v2
```

Comment:
It would be good to add this command to:
src/go/k8s/tests/e2e-v2/node-deleted/02-assert-delete-node-again.yaml
src/go/k8s/tests/e2e-v2/node-deleted/01-assert-delete-node.yaml